acceptancetests/repository/charms/appdata-source/hooks/leader-elected

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eux
+
+is_leader="$(is-leader)"
+if [ "$is_leader" = "False" ]; then
```

Comment:
I'm curious if we'd want something like per-unit tokens so that we can see that we take only the app token, but maybe not.

Is it worth something here that does some sort of "relation-get/set" as though it was the leader and sees that it gets an error (eg, trap the error, and return 0 from the hook, but make sure that we *get* the error)

Reply:
I like your idea of an action that tries to set the token - I don't think we need to be so careful about returning 0 like we would in a hook.